### PR TITLE
Change gui config and gui internal bitcoind directory architecture

### PR DESCRIPTION
**Pull request by edouardparis**
1. added the bitcoind parent directory
```
.liana/bitcoind
├── bitcoin-25.0
│   └── bin
└── datadir
    ├── anchors.dat
    ├── banlist.json
    ├── bitcoin.conf
    ├── blocks
    ├── chainstate
    ├── debug.log
    ├── fee_estimates.dat
    ├── mempool.dat
    ├── peers.dat
    └── settings.json
```

2. change gui config to have `start_internal_bitcoind` : `bool`

### CODE REVIEW

File: gui/src/installer/mod.rs [hunk 1]
```diff
@@ -324,7 +316,8 @@ pub async fn install(ctx: Context, signer: Arc<Mutex<Signer>>) -> Result<PathBuf
             daemon_config_path.canonicalize().map_err(|e| {
                 Error::Unexpected(format!("Failed to canonicalize daemon config path: {}", e))
             })?,
-            ctx.internal_bitcoind_exe_config.clone(),
+            // Installer started a bitcoind, it is expected that gui will start on on startup
```
Comment (darosior) [inline gui/src/installer/mod.rs:319]: nit

```suggestion
            // Installer started a bitcoind, it is expected that gui will start it on startup
```

**Comment by edouardparis** [comment]
> I'm just not sure about having this flag in the config instead of the settings

The gui will not be able to change it itself, it is setup only one time during install, so I think It can go in config.

**Comment by darosior** [comment]
Fair enough.

**Review by jp1ac4 — APPROVED** [review]
ACK https://github.com/wizardsardine/liana/pull/658/commits/bc87839e52fd64102ea24621e27cfcd26a32efd0

**Pull request merged**